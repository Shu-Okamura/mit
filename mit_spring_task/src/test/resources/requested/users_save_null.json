{
  "users": {
      "id": null,
      "name": null,
      "age": null,
      "tel": null,
      "addressInfo": {
        "addressid": null,
        "zipcode": null,
        "address": null
      }
    }
}