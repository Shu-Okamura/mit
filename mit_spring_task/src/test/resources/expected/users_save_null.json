{
  "code": 500,
  "fieldList": [
    {
      "name": "users.addressInfo.zipcode",
      "message": "must not be null"
    },
    {
      "name": "users.id",
      "message": "must not be null"
    },
    {
      "name": "users.addressInfo.address",
      "message": "must not be blank"
    },
    {
      "name": "users.name",
      "message": "must not be blank"
    },
    {
      "name": "users.age",
      "message": "must not be null"
    },
    {
      "name": "users.tel",
      "message": "must not be null"
    },
    {
      "name": "users.addressInfo.addressid",
      "message": "must not be null"
    }
  ]
}